stl/inc/functional

```diff
@@ -314,6 +314,7 @@ struct bit_not<void> {
     }
 };
 
+#if !_HAS_DEPRECATED_NEGATE

```

Comment:
Is this test reversed? (I suspect it was originally `#if !_HAS_CXX20` which would explain how this happened.)

Reply:
It indeed is